src/HashedDerivative.hs

```diff
@@ -708,14 +710,14 @@ mergeConstsBy op (exprs, nodes@(n1:_)) = (exprs', combined ++ nonconsts)
             (\n ->
                  case I.lookup n exprs of
                      Just (Const _ x) -> [x]
-                     _                -> [])
+                     _ -> [])
             nodes
     nonconsts =
         filter
             (\n ->
                  case I.lookup n exprs of
                      Just (Const _ _) -> False
-                     _                -> True)
+                     _ -> True)
```

Comment:
Perhaps this case shows an extreme case, but I still think it might be slightly better to have them lined up like the former.